CMakeLists.txt

```diff
@@ -75,6 +75,10 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake/config.hpp.in
     ${CMAKE_CURRENT_BINARY_DIR}/include/oomph/config.hpp @ONLY)
 install(FILES ${PROJECT_BINARY_DIR}/include/oomph/config.hpp
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/oomph)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake/cmake_config.hpp.in 
+    ${CMAKE_CURRENT_BINARY_DIR}/include/oomph/cmake_config.hpp)
```

Comment:
can we rename the resulting ending in .inc: `cmake_config.inc` in order to make it clear, that this is a code snippet?

Reply:
done